examples/example_MLP_model.cc

```diff
@@ -24,7 +24,7 @@
 #include <ATen/ATen.h>
 
```

Comment:
It would help a lot to name examples / tests that require CUDA with 'cuda' in their name (or in  a cuda subfolder), but this @doesn't have to be in the same PR, of course

Reply:
Will do, I barely managed to break the SDK dependence in the cmake and I am not very happy about the form it took.  Suggestions welcome.

Reply:
ideally, we want the same example to run on different backends in the future :)

Reply:
@ftynse , sure, but we need somehow to enable CPU only mode to run correctly for it :) May be we can use macro approach for it